Update source and target java version to 11

goneall opened the pull request:
Fixes #76

Signed-off-by: Gary O'Neall <gary@sourceauditor.com>

garydgregory commented:
Hi all,

Why not use the release property instead now that you require Java 11?

goneall commented:
> Why not use the release property instead now that you require Java 11?

@garydgregory I saw the suggestion in the Maven documentation, but when I added the property, it didn't build.

I replaced the previous source and target properties with:

```
<maven.compiler.release>11</maven.compiler.release>
```

and got the following error:

```
[ERROR] Source option 5 is no longer supported. Use 7 or later.
[ERROR] Target option 5 is no longer supported. Use 7 or later.
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

I'm probably doing something rather stupid - let me know if you see what the issue may be.

goneall commented:
I fixed the above issue by using the latest version of the maven compiler plugin.
